datanode/gateway/graphql/schema.graphql

```diff
@@ -2029,6 +2029,10 @@ type Party {
     assetId: ID
     "Optional pagination information"
     pagination: Pagination
+    "Optionally only return rewards >= this epoch"
+    fromEpoch: Int
+    "Optionally only return rewards <= this epoch"
+    toEpoch: Int
```

Comment:
**Nitpick (non-blocking)**: Ultra nitpick, but this is a uint64 in the GRPC, so I guess it’s technically possible that this *will eventually* overflow, if the epoch is set unreasonably short and the network lives for a long time. Hopefully by that point all browsers properly support big ints..

Regardless, I’m happy seeing an Int and not another string 😄

Reply:
yes fair enough, we use Int elsewhere for epoch though so at least it's consistent.. I'm going to just pretend it's never going to happen for now :)